packages/shared/src/components/cards/squad/SquadList.tsx

```diff
@@ -36,9 +36,9 @@ export const SquadList = ({
         alt={`${name} source`}
         type={ImageType.Squad}
       />
-      <div className="flex w-0 flex-grow flex-col">
+      <div className="flex max-w-[calc(100%-10rem)] flex-1 flex-col">
```

Comment:
This is a center element from a row, unless there is a specific width value, it won't truncate its content as it will never know the limits of the overflow.